.github/pull_request_template.md

```diff
@@ -41,6 +41,3 @@
 - [ ] The PR name is suitable for the release notes.
 - [ ] The solution is tested.
 - [ ] The documentation is up to date.
-- [ ] The PR has a priority label.
-- [ ] If the PR shouldn't be in the release notes, it has the
-      `C-exclude-from-changelog` label.
```

Comment:
What is the reasoning for deleting these?

Reply:
External contributors don't have access to the labels, so these items confuse them. If removing these doesn't work for us, let's bring them back.